IOS/SwiftSeedProject/SwiftSeedProject/Domain/Services/PersonService.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PersonService.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/28/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+open class PersonService {
+    let dataProvider : InMemoryDataProvider
+    
+    var persons = [Person]()
```

Comment:
This array should be declared as `let`.

Even if you modify it, you'll (and should) modify only the content, not the array itself (e.g. recreating it)